contrib/docker/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM gcr.io/distroless/base
+ARG BASE=gcr.io/distroless/base
+FROM $BASE
```

Comment:
nit: BASE is used only once,  so not sure it should be defined as an argument

Reply:
its an override-able argument with a default value. The travis.yml for ppc64 sets ```DOCKER_BASE``` to alpine (as gcr.io/distroless/base isn't multiarch (yet)).

Reply:
ok